pyrene/src/components/DateTimePicker/ReactDatePickerWrapper/ReactDatePickerWrapper.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useRef } from 'react';
+import ReactDatepicker from 'react-datepicker';
+import Icon from '../../Icon/Icon';
+
+import 'react-datepicker/dist/react-datepicker.css';
+import styles from '../datePicker.css';
+
+export interface DatePickerProps{
+  /**
+   * Close the calendar dropdown on date select
+   */
+  closeOnSelect?: boolean,
+  /**
+   * Replaces the input with any node, for example a button
+   */
+  customInput: React.ReactNode,
+  /**
+   * Date format used by component
+   */
+  dateFormat?: string,
+  /**
+   * Boolean to toggle time display
+   */
+  dateOnly?: boolean,
+  /**
+   * This is a Date object that represents the end date of a date range
+   */
+  endDate?: Date,
+  /**
+   * Display the component inline
+   */
+  inline?: boolean,
+  /**
+   * Calendar is opened on input component render
+   */
+  isOpen?: boolean,
+  /**
+   * This is a Date object that represents the maximum date allowed by the component
+   */
+  maxDate?: Date,
+  /**
+   * This is a Date object that represents the minimum date allowed by the component
+   */
+  minDate?: Date,
+  /**
+   * Do something when calendar opens
+   */
+  onCalendarOpen?: () => void,
+  /**
+   * Function to handle date change event
+   */
+  onChange?: (date: Date | [Date, Date] | null, event: React.SyntheticEvent<any> | undefined) => void,
```

Comment:
Try to reuse type instead of copying it :

`onChange?: ReactDatePickerProps['onChange']`

Reply:
✅